src/metrics/pipeline/applied/type.go

```diff
@@ -91,25 +91,25 @@ type OpUnion struct {
 
 // Equal determines whether two operation unions are equal.
 func (u OpUnion) Equal(other OpUnion) bool {
+	// keep in sync with Pipeline.Equal as go is terrible at inlining anything with a loop
 	if u.Type != other.Type {
 		return false
 	}
-	switch u.Type {
-	case pipeline.TransformationOpType:
```

Comment:
what about this case?

Reply:
The definition is `type TransformationOp struct{Type transformation.Type}`, where `transformation.Type` is plain `int`